src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -192,9 +192,16 @@ private static String getReasonForBeingLate(SyaCaseWrapper syaCaseWrapper) {
     }
 
     private static String getDwpIssuingOffice(SyaCaseWrapper syaCaseWrapper) {
-        String value = mrnIsNotProvided(syaCaseWrapper) ? null : syaCaseWrapper.getMrn().getDwpIssuingOffice();
-        return Norm.dwpIssuingOffice(
-            value);
+
+        if (syaCaseWrapper.getBenefitType().getCode().equalsIgnoreCase("uc")
```

Comment:
This assumes there will always be a benefit type

Reply:
There will from SYA